frontend/src/lib/components/shared/Modal/Modal.test.tsx

```diff
@@ -16,16 +16,20 @@
 
 import React from "react"
 import { BaseProvider, LightTheme } from "baseui"
+import "@testing-library/jest-dom"
+import { render } from "src/lib/test_util"
 
-import { mount } from "src/lib/test_util"
 import Modal from "./Modal"
 
 describe("Modal component", () => {
   it("renders without crashing", () => {
-    mount(
+    render(
       <BaseProvider theme={LightTheme}>
         <Modal isOpen />
       </BaseProvider>
     )
+
+    const modalElement = document.body.querySelector('[role="dialog"]')
+    expect(modalElement).toBeInTheDocument()
```

Comment:
Perhaps for consistency we could use something like this?
```javascript
const modalElement = screen.getByRole('dialog')
expect(modalElement).toBeInTheDocument()
```

Reply:
Yes!